### PR TITLE
Update pbs_exporter.sh

### PR DESCRIPTION
**Pull request by ebuzzz**
I've added a couple of improvements for our local use, which I want to share.

feat: add support for namespaces
feat: add support for fetching task statuses
feat: retrieve estimated full date
feat: add support for custom PBS_CURL_OPTIONS via configuration

### CODE REVIEW

**Comment by ebuzzz** [comment]
If this is something that you want to merge, I can do some work on the PR (documentation, moving the hardcoded --insecure to a separate config option).

**Comment by rare-magma** [comment]
@ebuzzz thanks for your contribution, this looks interesting so I think it's worth investing in what you mention (documentation + config option for `--insecure` before merging. I can review it later today and get back to you then.

File: pbs_exporter.sh
Comment (rare-magma) [inline pbs_exporter.sh]: @ebuzzz this can be simplified to something like:
`twenty_four_hours_ago=$(date +%s --date "24 hours ago")`

File: pbs_exporter.sh
Comment (rare-magma) [inline pbs_exporter.sh:30]: @ebuzzz which additional curl options are useful for your use case? 
If it's just the `--insecure` flag then I'd suggest creating a new configuration option such as `INSECURE_SSL_VERIFICATION` so that it is obvious that it results in a lower security level

Comment (rare-magma) [reply pbs_exporter.sh:30]: Reducing repetition by setting all common flags in one variable is a good idea tho

File: pbs_exporter.sh
Comment (rare-magma) [inline pbs_exporter.sh:165]: @ebuzzz I think namespace is missing here

**Review by rare-magma — CHANGES_REQUESTED** [review]

**Comment by rare-magma** [comment]
@ebuzzz I've tested this on my setup (which has no namespaces configured) and it seems to work so I count on you to confirm the data is correct on such setups

File: pbs_exporter.sh
Comment (rare-magma) [reply pbs_exporter.sh:165]: nevermind I see that there is no namespace breakdown on the snapshot count

File: pbs_exporter.sh
Comment (ebuzzz) [reply pbs_exporter.sh:30]: For now, only insecure is needed for my use-case. But I could think of use for `--capath`, `--ciphers`. They won't be needed often, but sometimes internal certificates can be messy.

I'll let you decide. If you want a single option for the insecure, I'll modify it that way.

Comment (ebuzzz) [reply pbs_exporter.sh:30]: We can also use both: `INSECURE_SSL_VERIFICATION` for both the PBS and PushGateway cURL calls. And a `CURL_OPTIONS`  for all the other options?

File: README.md
Comment (rare-magma) [inline README.md:127]: @ebuzzz shouldn't it be `PBS_TASK_PERIOD_HOURS` instead? Or you could change the date variable to days if you prefer: 
`twenty_four_hours_ago=$(date +%s --date "${PBS_TASK_PERIOD_DAYS} days ago")`

File: pbs_exporter.sh
Comment (rare-magma) [reply pbs_exporter.sh:30]: I see, thanks for clarifying!
I think the 2 variable approach (INSECURE_SSL_VERIFICATION +CURL_OPTIONS) sounds good

**Review by rare-magma — CHANGES_REQUESTED** [review]

File: README.md
Comment (rare-magma) [reply README.md:127]: notice `... days ago` instead of `... hours ago`

**Review by rare-magma — CHANGES_REQUESTED** [review]

File: README.md
Comment (rare-magma) [reply README.md:127]: notice `... days ago`  instead of `... hours ago`

**Comment by rare-magma** [comment]
@ebuzzz are you still interested in merging this?

**Comment by ebuzzz** [comment]
@rare-magma Yes, sorry for the delay. I'm currently working on a different project, but will pick it up as soon as possible.

**Comment by rare-magma** [comment]
@ebuzzz thanks for your contribution. I no longer use proxmox backup server so development of this tool has ceased. Feel free to fork it.

**Pull request closed without merging**
